index.bs

```diff
@@ -1036,6 +1036,10 @@ An <dfn>attribution rate-limit record</dfn> is a [=struct=] with the following i
 :: A [=moment=].
 : <dfn>expiry time</dfn>
 :: Null or a [=moment=].
+: <dfn>entity ID</dfn>
+:: Null or an ID.
+: <dfn>deactivated</dfn> (default false)
```

Comment:
The spec to this point has avoided the concept of source activation. Is it really necessary now?

Reply:
It's a different concept of "deactivation", it's to deactivate a source rate-limit record for the purpose of couting the distinct destinations for unexpired sources.

Reply:
Should we give the field a more specific name if it's only deactivated with respect to a particular limit? Otherwise it seems like every rate-limit algorithm needs to inspect this field.

Reply:
Yeah I’m thinking about a more specific name as well, like “deactivated for unique destination limit for unexpired sources”. Please feel free to suggest a better name if any. Thanks.